no-nuxt-pls/components/NewsItem/NewsItem.vue

```diff
@@ -2,13 +2,32 @@
   <div>
     <h2>{{news.title}} ({{news.votes}})</h2>
     <br>
-    <button @click="updateNews(1)">Upvote</button>
-    <button @click="updateNews(-1)">Downvote</button>
-    <button @click="deleteNews">Remove</button>
+    <button v-if="token" @click="updateNews(1)">Upvote</button>
```

Comment:
star For an upvote button that behaves according to the authentication state of your user